graphics/ZBar/Portfile

```
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    ZBar
+github.setup            mchehab zbar 0.23.90
+revision                0
+
+checksums               rmd160  ce5fb3bb7376ae0208cdb83baf706d447a11dfe5 \
+                        sha256  9581b5ab467f7b45ef0d7219e773c8dd0de07eb916f32f6828cd48b96ce50344 \
+                        size    1005439
+
+categories              graphics
+maintainers             {@knapoc knapoc} openmaintainer
+
+description             ZBar is an open source software suite for reading bar codes from \
+                        various sources, including webcams.
+long_description        ZBar Bar Code Reader is an open source software suite for reading \
+                        bar codes from various sources, such as video streams, image files \
+                        and raw intensity sensors. It supports EAN-13/UPC-A, UPC-E, EAN-8, \
+                        Code 128, Code 93, Code 39, Codabar, Interleaved 2 of 5, QR Code and SQ Code.
+
+platforms               darwin
+license                 LGPL-2.1
+
+depends_lib             port:GraphicsMagick
+
+use_autoreconf          yes
+autoreconf.args         --install --verbose --force
+configure.cmd-append    -disable-video --disable-nls --with-python=python3
```

```suggestion
configure.args-append   --disable-video --disable-nls
```

```
  configure: error: Python module enabled, but unable to compile Python.h.
  Install the development package for python-3.8, or configure
  --without-python to disable the python bindings.
```

Python needs to be set as a lib dependency, and you should set the path to MacPorts' Python. Maybe it would be useful to set the python bindings as an optional variant?

e.g.

```
variant python39 description {Include Python 3.9 binding} {
    configure.args-append --with-python=${prefix}/bin/python3.9
    depends_lib-append port:python39
}
```

---

Thanks for the hint.. I implemented your variant suggestion but used `configure.env-append PYTHON=${prefix}/bin/python3.9` instead because `--with-python` only accepts values of `no`, `auto`, `python2` and `python3`.